templates/report_template.html

```diff
@@ -22,23 +22,35 @@
 					<td>
 						<p><b>Summary:</b></p>
 						<ul>
-							<li>{{summary.total_success}}</li>
-							<li>{{summary.total_unstable}}</li>
-							<li>{{summary.total_failure}}</li>
+							<li>Total {{summary.total_success}}</li>
+							<li>Total {{summary.total_unstable}}</li>
+							<li>Total {{summary.total_failure}}</li>
 							{% if summary.total_missing %}
-							<li>{{summary.total_missing}}</li>
+							<li>Total {{summary.total_missing}}</li>
 							{% endif %}
 							{% if summary.total_aborted %}
-							<li>{{summary.total_aborted}}</li>
+							<li>Total {{summary.total_aborted}}</li>
 							{% endif %}
 							{% if summary.total_error %}
-							<li>{{summary.total_error}}</li>
+							<li>Total {{summary.total_error}}</li>
 							{% endif %}
 							<li>{{summary.total_bugs}}</li>
 							<li>{{summary.total_tickets}}</li>
-							<li>{{summary.total_coverage}}</li>
+							<li>Total {{summary.total_coverage}}</li>
 						</ul>
 					</td>
+					{% for sorted_version in stats_per_version.keys()|sort|reverse %}
+					<td>
+						<p><b>Summary for {{ (sorted_version|string).rstrip('.0') }}</b></p>
```

Comment:
@nathan-weinberg 13 is addressed here already
![after-fix](https://user-images.githubusercontent.com/20891964/112203470-db364900-8c12-11eb-9f38-9de9c643b43f.png)

Reply:
I'm talking about in the report table

Reply:
Sorry, my misunderstanding. It is fixed now both in the table and summary.